flake.nix

```diff
@@ -1,31 +1,35 @@
 {
   description = "Lambda Buffers";
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs";
+    nixpkgs.follows = "flake-lang/nixpkgs";
 
     # flake-lang.nix used for monorepo setups
     flake-lang.url = "github:mlabs-haskell/flake-lang.nix";
 
     # Flakes as modules, using this extensively to organize the repo into modules (build.nix files)
-    flake-parts.url = "github:hercules-ci/flake-parts";
+    flake-parts.follows = "flake-lang/flake-parts";
 
     # Code quality automation
-    pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
+    pre-commit-hooks.follows = "flake-lang/pre-commit-hooks";
 
     # Hercules CI effects
-    hci-effects.url = "github:hercules-ci/hercules-ci-effects";
+    hci-effects.follows = "flake-lang/hci-effects";
 
     # Nix library for Google Protobufs
-    proto-nix.url = "github:mlabs-haskell/proto.nix";
+    proto-nix = {
+      url = "github:mlabs-haskell/proto.nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+      inputs.haskell-nix.follows = "flake-lang/haskell-nix";
```

Comment:
I suppose we could make _all_ of [proto.nix](https://github.com/mlabs-haskell/proto.nix/blob/main/flake.nix)'s inputs follow our stuff, but I don't think it's significant enough of a difference to warrant this change.